src/main/java/teammates/logic/api/Logic.java

```diff
@@ -439,13 +439,13 @@ public InstructorAttributes getInstructorForRegistrationKey(String encryptedKey)
      * @return null if the key doesn't exist.
      * @throws EntityDoesNotExistException
      */
-    public String getKeyForInstructor(String courseId, String email)
+    public String getEncryptedKeyForInstructor(String courseId, String email)
```

Comment:
Change the javadoc for this, it says `Get the decrypted registration key` currently.